modules/ROOT/pages/deployment/storage/posixfs.adoc

```diff
@@ -0,0 +1,104 @@
+====
+
+== PosixFS Overview
+
+When PosixFS is configured, a user may not only access data via Infinite Scale as usual, but also via the filesysem directly. This concept allows, for example, services like printers, scanners to write data to a location Infinite Scale has access to and treats these files like they were managed thru Infinite Scale directly. On the other hand, users or services can access data for further processing as they would normally do via the filesystem without going thru Infinite Scale.
+
+Comparison of private vs shared storage in Infinite Scale::
+* When *private storage* is configured: +
+Only Infinite Scale can manage data. It is in full control over the data and structure, which is not easily human readable.
+* When *shared storage* is configured: +
+Infinite Scale is not in full control over the underlying data and structure. It needs to be notified of changes and has to add required metatdata via extended attributes to make the process flow work like in a private storage.
+
+== Limitations
+
+* A filesystem must be posix compliant and must fully support extended attributes.
+* Only inotify and GPFS file system change notification methods are supported. +
+For mounted filesystems, `inotifywait` must be installed.
+* Versioning is not supported yet.
+* The spaces are folders in the filesystem and named after the UUID, not the real space name
+* No support for CephFS or other mounted filesystems like NFSv3 or SMB yet. +
+NFSv4.2 is working. This filesystem provides sufficient extended attribute capabilities. Check the documentation of your storage provider for details.
+* Postprocessing like antivirus scanning does not get triggered for file actions that are not initiated by Infinite Scale.
+* Though there are more options available, we recommend using a group based access for the shared storage. This means, that both Infinite Scale and the accessing users/services must use a common group. The default umask in the directory used has to allow group writing all over the place.
+
+== Setup
+
+Note that it is possible to use different storage drivers in the same Infinite Scale installation. For example, one is set up for one space running on PosixFS Storage Driver while all others run Decomposed FS with a private storage. This is an enhanced configuration and requires multiple instances of the xref:{s-path}/storage-users.adoc[Storage-Users] service running, individually configured. The details are not covered here.
+
+=== Prerequisites
+
+To use the PosixFS Storage Driver, the following prerequisites have to be fulfilled:
+
+* There must be storage available to store meta data and blobs, available under a root path.
+* When using inotify, the storage must be local on the same machine. +
+Network mounts require `inotifywait` to be installed (inotify does not work here).
```

Comment:
This is not really correct. `inotifywait` is a hard requirement for the inotify watcher, regardless if it's a mounted remote filesystem or a local one.